orders.js

```diff
@@ -0,0 +1,89 @@
+const createOrder = async (pool, { datePlaced, status, customerId }) => {
+    const {rows:[order]} = await pool.query(`
+        INSERT INTO orders(
+            date_placed,
+            status,
+            customer_id
+        ) VALUES ($1, $2, $3)
+        RETURNING *;   
+    `, [datePlaced, status, customerId]);
+    return order;
+}
+
+const getAllOrders = async (pool) => {
+    const {rows: orders} = await pool.query(`
+        SELECT * FROM orders;
+    `)
+    return orders;
+}
+
+const getOrderById = async (pool, id) => {
+    const {rows:[order]} = await pool.query(`
+        SELECT * FROM orders
+        WHERE id = $1;
+    `, [id])
+    if (order === undefined) {
+        throw new Error(`Could not retrieve data because id provided (${id}) does not exist in table.`)
+    } 
+    return order;
+}
+
+const getCustomerOrders = async (pool, customerId) => {
```

Comment:
```suggestion
const getOrdersByCustomerId = async (pool, customerId) => {
```

Reply:
This is especially important because it could lead to confusion and make someone think you are referring to a join table